ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/PrivacyParameters.java

```diff
@@ -201,6 +204,16 @@ public Builder setMultiTenancyEnabled(final boolean multiTenancyEnabled) {
       return this;
     }
 
+    public Builder setTlsConfiguration(
+        final Path privacyKeyStoreFile,
+        final Path privacyKeyStorePasswordFile,
+        final Path privacyClientWhitelistFile) {
```

Comment:
update this param and field to privacyServersWhitelistFile as that's what it's called in BesuCommand